translations/Arabic/special-terms.md

```diff
@@ -0,0 +1,48 @@
+# Terms
+
+- Message: رسالة
+  لقد وصلتك رسالة
+- Private Message (PM): رسالة خاصة
+  أرسل رسالة خاصة
+- Group PM: رسالة خاصة لمجموعة
+- Realm: عالم
+  Out of context, it would mean 'kingdom', so the word for 'world' is better here
```

Comment:
Over here 'realm' is used in the context of domain so maybe we can use an Arabic word for domain?